nodes.py

```diff
@@ -1419,7 +1419,8 @@ def __init__(self):
     def INPUT_TYPES(s):
         return {"required": 
                     {"images": ("IMAGE", ),
-                     "filename_prefix": ("STRING", {"default": "ComfyUI"})},
+                     "filename_prefix": ("STRING", {"default": "ComfyUI"}),
+                     "disable_metadata": ("BOOLEAN", {"default": False})},
```

Comment:
You would want to add this under `optional` to avoid breaking presaved API workflows